easybuild/easyconfigs/g/Gaussian/Gaussian-09.e.01-AVX.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
+
+name = 'Gaussian'
+version = '09.e.01'
+versionsuffix = '-AVX'
+
+homepage = 'https://www.gaussian.com/'
+
+# For the sake of users I advise changing the last line in the description
+# and versionsuffix to match the actual installed version.
+description = """
+Gaussian provides state-of-the-art capabilities for electronic structure
+modeling. Gaussian 09 is licensed for a wide variety of computer
+systems. All versions of Gaussian 09 contain every scientific/modeling
+feature, and none imposes any artificial limitations on calculations
+other than your computing resources and patience.
+
+This is the official gaussian AVX build.
+"""
+
+toolchain = SYSTEM
+
+# Gaussian is licensed software.
+# You need to pick the tar file of your choice from the DVD,
+# put it in your sources tree (should be a tree that ordinary users can't read)
+# and change the name in "sources"
+sources = ['09.e.01-AVX-A64-114X.tgz']
+checksums = ['af1e1cdf7d2799690a642b27ee0779b7cca9695b9a86a48739f3d9fc8c54a98d']
+
+start_dir = '..'
+
+postinstallcmds = [
+    'mkdir %(installdir)s/bin',
+    'ln -s ../g%(version_major)s/g%(version_major)s ../g%(version_major)s/ghelp %(installdir)s/bin',
```

Comment:
Is it worth adding a sanity check command that does `g09 --version` or something? Not sure if that's a viable option here...

Reply:
It doesn't have a --version, or -v, or -V or basically anything useful for a sanity check.

Reply:
ok, fine